reference/performance/statement-summary.md

```diff
@@ -123,28 +230,43 @@ The statement summary feature is disabled by default. You can enable it by setti
-
-`events_statements_summary_by_digest` is a memory table. To prevent potential memory issues, we need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under `[stmt-summary]` of `config.toml`:
+The statement summary tables are memory tables. To prevent potential memory issues, we need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under `[stmt-summary]` of `config.toml`:
 
-- `max-stmt-count` limits the number of SQL categories that can be saved. The default value is 100. If the set limit is exceeded, the most recent SQL statements that remain unused will be removed.
+- `max-stmt-count` limits the number of SQL statements that can be stored. The default value is `200`. If the set limit is exceeded, those SQL statements that recently remain unused will be cleared.
 - `max-sql-length` specifies the longest display length of `DIGEST_TEXT` and `QUERY_SAMPLE_TEXT`. The default value is 4096.
 
 > **Note:**
 >
-> It is suggested that you adjust the configuration of this two parameters based on your actual scenarios. However, setting them to too large values is not recommended.
+> The `tidb_stmt_summary_history_size`, `max-stmt-count`, and `max-sql-length` configuration items affect memory usage. It is recommended that you adjust these configurations based on your actual needs. It is not recommended to set them too large values.
 
 ## Known limitations
 
-`events_statements_summary_by_digest` has some known limitations:
+The Statement summary tables have some known limitations:
 
-- Querying `events_statements_summary_by_digest` only returns the statement summary of the current TiDB server, not the entire cluster.
-- Statement summary tables do not support rolling update. This means that the SQL statistics start to accumulate the second `tidb_enable_stmt_summary` is enabled. As time goes, the numbers mount. This makes it impossible to view the statement summary of the most recent period. Therefore, it's strongly recommended that you enable `tidb_enable_stmt_summary` when you need to troubleshoot issues.
-- The statement summary will be lost when the TiDB server restarts. This is because `events_statements_summary_by_digest` is a memory table, and the data is cached in memory instead of persisted on storage.
+- Querying statement summary tables only returns the statement summary of the current TiDB server, not that of the entire cluster.
+- The statement summary will be lost when the TiDB server restarts. This is because statement summary tables are memory tables, and the data is cached in memory instead of persisted on storage.
```

Comment:
```suggestion
- The statement summary will be lost when the TiDB server restarts. This is because statement summary tables are memory tables, and the data is cached in memory instead of being persisted on storage.
```